README.md

```diff
@@ -79,6 +79,56 @@ cd corral
 
 You can make a debug build with `.\make.ps1 build -Config Debug`
 
+## Getting started using Corral
+
+After instllation, make sure Corral's current path is also set in your PATH environment variable. This will make it easier by not entering the full path each time when you use one of Corral commands in your projects.
+
+Follow these steps to create your first project using Corral.
+
+1. Create an empty folder and switch to this directory. This will be our example project to use Corral
+
+```bash
+mkdir myproject
+cd myproject
+```
+
+2. Initialize Corral. It will create `corral.json` and `lock.json` files. At this moment they won't have much information since you haven't added any dependency yet.
+
+```bash
+corral init
+```
+
+3. Add a dependency. This is the way to tell Corral that your project depends on this and you want to include it when building your project.
+
+```bash
+corral add github.com/jemc/pony-inspect.git
+```
+
+4. Use that dependency in your code. Create a file `main.pony` with following code.
+
+```pony
+use "inspect"
+
+actor Main
+  new create(env: Env) =>
+    env.out.print(Inspect("Hello, World!"))
+```
+
+5. Fetch dependencies. The example Pony code is using `Inspect` type which is defined in the dependency which you have just added. Pony needs to have the source code of `Inspect` type to compile successfully. By fetching, Corral retrieves the source and make it available when compiling the source code 
+
+```bash
+corral fetch
+```
+
+6. Build the project. Corral will now use this information to build the project.
```

Comment:
Another consistency point referring back to "Create a project. ..." again. It should either be "Create a project" and here "Build a project" or "Create the project" and "Build the project" so the article matches in specificity. I vote "the" even though I wrote differently above in "Create a project."